Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/Properties/ParameterExtractor.cs

```diff
@@ -218,6 +220,15 @@ ILogger<ParameterExtractor> logger
           return null;
         }
 
+        // "SYMBOL_ID_PARAM" is internal name for "Type ID". localization impacts definition names, internal more reliable
+        // probably a waste to reuse HandleDefinition method here? simple approach would be to change method signature for GetValue to also take
+        // internalDefinitionName as argument. we're discarding most returns here anyway
+        var (internalDefinitionName, _, _, _) = _parameterDefinitionHandler.HandleDefinition(parameter);
```

Comment:
@clairekuang using `HandleDefinition` makes sense to me, but it has already been called in `ParseParameterSet` (which then calls this `GetValue` method).

I'm inclined to change the method signature for `GetValue` to include the `internalDefinitionName` as an arg, so we don't need to recall it here?